Scenarios/TripPin/src/webapi/ODataSamples.WebApiService/Helper/ODataExtession.cs

```diff
@@ -3,10 +3,12 @@
     using System;
     using System.Linq;
     using System.Net.Http;
+    using System.Web.Http;
     using System.Web.Http.Routing;
-    using System.Web.OData.Extensions;
-    using System.Web.OData.Routing;
-    using Microsoft.OData.Core.UriParser;
+    using Microsoft.AspNet.OData.Extensions;
+    using Microsoft.AspNet.OData.Routing;
+    using Microsoft.OData;
+    using Microsoft.OData.UriParser;
 
     public static class ODataExtession
```

Comment:
nit: This should be ODataExtension.

Reply:
Yep; I know. I think the examples need a full pass, but decided to keep this update primarily to moving to the latest WebAPI and fixing the broken nav prop bindings.